CONTRIBUTING.pt_br.md

```diff
@@ -77,7 +77,7 @@ Infelizmente, seu editor de texto não o terá. Portanto, existem extensões par
 
 ## Formatando
 
-**Nota:** GitHub e as três extensões listadas acima garantirão que você estará usando a formatação correta automaticamente.
+**Nota:** GitHub e às três extensões listadas acima garantirão que você estará usando a formatação correta automaticamente.
```

Comment:
this change is incorrect and translates as `Github and TO the three extensions listed above` instead of the correct original one `Github and the three extensions listed above`